transposon/submit.sh

```diff
@@ -52,14 +52,21 @@ bclen=$6
 BCreadSeq=$7
 plasmidSeq=$8
 extractBCargs=$9
-
+analyzeBCargs=""
 shift 9
+
+while getopts ':b:' opt; do
```

Comment:
the initial : suppresses errors, right? why?

Reply:
you mean it will fail on the remaining arguments (ie samples)?

Reply:
No. If I include and options different of `-b`, it prints an error message like "unrecognized option" and continues execution. I didn't try including `-b` and a non-recognize option to see if it fails to recognize -b